src/CodeGen_LLVM.cpp

```diff
@@ -307,7 +314,12 @@ CodeGen_LLVM *CodeGen_LLVM::new_for_target(const Target &target,
         return make_codegen<CodeGen_PowerPC>(target, context);
     } else if (target.arch == Target::PNaCl) {
         return make_codegen<CodeGen_PNaCl>(target, context);
+#ifdef WITH_HEXAGON
```

Comment:
I don't think that solves the problem, which is distinguishing between a symbol being not defined vs. being defined but equal to 0.